SusyCommon/XaodAnalysis.h

```diff
@@ -325,6 +341,7 @@ namespace Susy {
     bool matchTruthJet(int iJet); ///< Match a reco jet to a truth jet
 
     bool eleIsOfType(const xAOD::Electron &in, eleID id=eleID::LooseLLH);
+    bool muIsOfType(const xAOD::Muon &in, muID id=muID::Medium);
```

Comment:
I agree. This function is to check for a specific type, so calling without the type argument would be out of sync with what this function is meant for.